admin_manual/configuration_files/serving_static_files_configuration.rst

```diff
@@ -130,6 +130,26 @@ Configuration is similar to Apache::
 
   * **~ ^/data/[^/]+/files/LOCAL-MOUNT-NAME/(.+)$** ~ a local external storage mount available to all users
 
+Configuration - Method 2
+~~~~~~~~~~~~~~~~~~~~~~~~
+Method 2 is simplier to setup when using local external storage mounts, especially when they are available to many, but not all users.
+This method may be prefered if you are regularly adding users that should not all have access to the same local external storage mount(s). ::
```

Comment:
"simplier" should be "simpler". I can't speak to the technical accuracy, but  looks good, clear and understandable.